api/src/main/java/org/openmrs/module/clientregistry/providers/r4/FhirCRPatientResourceProvider.java

```diff
@@ -91,7 +91,8 @@ public List<Patient> getCRPatientById(
 		return patients;
 	}
 	
-	@Search
+
```

Comment:
Do you need this extra line break? Don't forget to remove the `ca.uhn.fhir.rest.annotation.Search` import statement.

Reply:
Done